app/serializers/ticket_serializer.rb

```diff
@@ -29,8 +29,9 @@ def ask_for_update
     Ticket::ASK_FOR_UPDATE
   end
 
-  def permited_transitions
+   def permited_transitions
     status = object.status
+    role = scope&.role&.name
```

Comment:
Why do we need to use scopes?, ideally serializers should be independent of logged in user, you should change code calling this serializer to fit your requirement. However, as a quick fix, can we pass current user from controller as a argument only for this serializer.

Reply:
i had tried the same to pass through controller but its getting null I will check again